WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift

```diff
@@ -24,7 +24,7 @@ final class QuickStartPromptViewController: UIViewController {
 
     /// Closure to be executed upon dismissal.
     ///
-    var onDismiss: ((Blog) -> Void)?
+    var onDismiss: ((Blog, Bool) -> Void)?
```

Comment:
Thanks for fixing this closure! Can you add a comment to clarify what the params mean?

Reply:
Done in c329da329787912562596f06045f6c17d1130b65